src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -95,6 +97,9 @@ def __init__(self, msConfig, logger=None):
         self.rseConsStats = {}
         self.protectedLFNs = []
 
+        # The basic /store/unmerged regular expression:
+        self.regStoreUnmerged = re.compile("^/store/unmerged/.*$")
```

Comment:
Should we ensure that the path to be deleted will always match a given depth in the directory structure?
E.g., something like `re.compile("^/store/unmerged/*/*/*/.*$")`?

Reply:
I'd rather say no. Because there are stuff that are recorded like SAM tests outputs and so on which are certainly not reaching the depth to what we'd define here... and even from our workflows [1]. In the expression you suggest I suspect are mixed wildcards + regexp, but anyway it can be worked out of course I've got the idea. And actually initially I did try with the following regular expression `r'^/store/unmerged/.+/.+/.+/.+$'` which should be the one expressing the depth correctly, but if yout think twice it is exactly equivalent to just this one:  `'^/store/unmerged/.+$'. So things more or less lead to the generic one. For dealing with the path depth there is the dictionary slicing `[:6]` down the code, which should always provide us the maximum depth to be reached. 


[1]
```
'/store/unmerged/Phase2HLTTDRSummer20ReRECOMiniAOD/DYToLL_M-50_TuneCP5_14TeV-pythia8/FEVT',
'/store/unmerged/Phase2Spring21DRMiniAOD/TTbar_TuneCP5_14TeV-pythia8/FEVTDEBUGHLT',
'/store/unmerged/Phase2Spring21DRMiniAOD/TTbar_TuneCP5_14TeV-pythia8/MINIAODSIM',
'/store/unmerged/Run2016B/BTagCSV/MINIAOD',
'/store/unmerged/Run2016B/BTagMu/MINIAOD',
'/store/unmerged/Run2016B/DoubleEG/AOD',
'/store/unmerged/Run2016B/DoubleEG/MINIAOD',
'/store/unmerged/Run2016B/DoubleMuonLowMass/MINIAOD',
'/store/unmerged/Run2016B/HTMHT/MINIAOD',                           
  ...
 '/store/unmerged/RunIISummer20UL18wmLHEGEN/ttHJetToNonbb_M125_TuneCP5_13TeV_amcatnloFXFX_madspin_pythia8/GEN',
 '/store/unmerged/RunIISummer20UL18wmLHEGEN/ttHToNonbb_M125_TuneCP5_13TeV-powheg-pythia8/GEN',
 '/store/unmerged/RunIISummer20UL18wmLHEGEN/ttH_HToInvisible_M125_TuneCP5_13TeV_powheg_pythia8/GEN',
 '/store/unmerged/SAM/testSRM/SAM-cms-lvs-gridftp.hep.wisc.edu',
 '/store/unmerged/SAM/testSRM/SAM-cmssrm.hep.wisc.edu',
 '/store/unmerged/SnowmassWinter21GEN/GJet_Pt-20to40_DoubleEMEnriched_MGG-8toInf_TuneCP5_14TeV-pythia8/GEN'
```

Reply:
And actually I just spotted I was cutting the path one level higher than I should have. Fixing it with my next commit.

Reply:
fixed